README.md

```diff
@@ -1 +1,29 @@
-# team-culture
\ No newline at end of file
+# Code of Conduct
+
+## Diversity
+We accept others, no matter the gender, nationality, culture, religion.
+
+## Respect
+We all have different backgrounds, beliefs and points of view.
+Treat others as you wish to be treated.
+
+## Trust and Cooperation
+We are a team, don't be afraid to be judged. We support each other.
+
+## Autonomy
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
+
+## Responsability
+You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.
+
+## Ownership
+Be the owner of your code and projects.  If something fails on it, you have to take care of it.
+
+## Disagree
```

Comment:
## Healthy Culture of Controversy
In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Additionally the person who works on the task has the right to stop the argument at any time, take an informed decision and continue working.